checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/api/PactTest.kt

```diff
@@ -294,6 +296,62 @@ class PactTest {
             .toPact()
     }
 
+    @Pact(provider = "verified-tokens", consumer = "access-checkout-android-sdk")
+    fun createEmptyBodyErrorInteractionRequestInteraction(builder: PactDslWithProvider): RequestResponsePact {
+        return builder
+            .uponReceiving("A request for a session reference with empty body")
+            .path(path)
+            .method("POST")
+            .headers("Content-Type", "application/vnd.worldpay.verified-tokens-v1.hal+json")
+            .headers("Accept", "application/vnd.worldpay.verified-tokens-v1.hal+json")
+            .body("")
+            .willRespondWith()
+            .status(400)
+            .headers(
+                mapOf(
+                    Pair(
+                        "Content-Type",
+                        "application/vnd.worldpay.verified-tokens-v1.hal+json;charset=UTF-8"
+                    )
+                )
+            )
+            .body(
+                generateResponseVariation1(
+                    "bodyIsEmpty",
+                    "The body within the request is empty"
+                )
+            )
+            .toPact()
+    }
+
+    @Pact(provider = "verified-tokens", consumer = "access-checkout-android-sdk")
+    fun createVariation2ErrorInteraction(builder: PactDslWithProvider): RequestResponsePact {
```

Comment:
Aren't these two interactions the same @hcmsilva ?

Reply:
👍 yup it wasn't even in use, removed.